net-misc/gallery-dl/gallery-dl-1.18.1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python3_{8,9})
+PYTHON_REQ_USE="sqlite,ssl,xml"
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1 optfeature
+
+DESCRIPTION="Download image galleries and collections from several image hosting sites"
+HOMEPAGE="https://github.com/mikf/gallery-dl"
+
+if [[ ${PV} == *9999 ]]; then
```

Comment:
Trivial but I prefer to avoid `*` if not going to make use of it, this is always exactly 9999